LondonBritishLibrary/add/BLadd19341.xml

```diff
@@ -69,7 +69,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
-                                    <material key="paper"/> wich bears <watermark>the date
+                                    <material key="paper"/> which bears <watermark>the date
```

Comment:
the visualization at the moment does not support this sort of encoding, 
"paper" is printed as a button, not part of text;
"watermark" is printed as yes/no; date is not parsed

If this needs to be fixed a separate issue would be needed